spec/models/task_spec.rb

```diff
@@ -9,4 +9,19 @@
       expect(task).to_not be_valid
     end
   end
+
+  context 'with a name but no user' do
+    it 'is invalid' do
+      task = Task.new(name: "First task")
+      expect(task).to_not be_valid
+    end
+  end
+
+  context 'with a name and a user' do
+    it 'is valid' do
+      user = User.new(email: "email@gmail.com", password: "password")
+      task = Task.new(name: "First task", user: user)
+      expect(task).to be_valid
+    end
+  end
```

Comment:
womp womp womp... there are some critical features that need to be tested, particularly the index and create/update actions, but i ran out of time.